### PR TITLE
 Get source from *vm.Program

### PR DESCRIPTION
**Pull request by 100vlx**
Get  source from *vm.Program

### CODE REVIEW

**Comment by 100vlx** [comment]
link: https://github.com/expr-lang/expr/discussions/489

File: vm/program.go [hunk 1]
```diff
@@ -343,3 +343,8 @@ func (program *Program) DisassembleWriter(w io.Writer) {
 		}
 	}
 }
+
+// SourceContent returns origin string.
+func (program *Program) SourceContent() string {
```
Comment (antonmedv) [inline vm/program.go:348]: What about just `Source()`?

Comment (100vlx) [reply vm/program.go:348]: Yes I need this field

Comment (100vlx) [reply vm/program.go]: ok, I have already edited it

**Review by antonmedv — APPROVED** [review]

**Comment by 100vlx** [comment]
Did fuzzing go wrong?

**Comment by 100vlx** [comment]
Please deal with this, thank you

**Pull request merged**